src/nuget/xunit.console.netcore.nuspec

```diff
@@ -33,11 +33,21 @@
       <dependency id="System.Threading" version="4.0.10" />
       <dependency id="System.Threading.Tasks" version="4.0.10" />
       <dependency id="System.Xml.XDocument" version="4.0.10" />
+      <dependency id="xunit.runner.utility" version="2.1.0-rc1-build3168" />
     </dependencies>
   </metadata>
   <files>
     <!-- xunit runner for .NET Core -->
     <file src="xunit.console.netcore\xunit.console.netcore.exe" target="lib\aspnetcore50" />
+    <file src="xunit.console.netcore\_._" target="lib\dnxcore50" />
+    <file src="xunit.console.netcore\_._" target="lib\dotnet" />
+    
+    <!--
+      NuGet ignores .exe files in lib folders, so use .props files to copy the exe
+      to the output folder.
+    -->
+    <file src="xunit.console.netcore\xunit.console.netcore.exe" target="build\dotnet" />
+    <file src="xunit.console.netcore\xunit.console.netcore.props" target="build\dotnet" />
   </files>
```

Comment:
You don't have to modify your project files but you do end up with a *.nuget.props/targets file in every directory for projects that consume these. I would prefer if we can avoid those and instead include this copy logic in a common test.targets from build tools, assuming you are using build tools package.